app/services/distribution_update_service.rb

```diff
@@ -12,7 +12,7 @@ def call
       @old_delivery_method = distribution.delivery_method
       distribution.storage_location.increase_inventory(distribution.to_a)
       # Delete the line items -- they'll be replaced later
-      distribution.line_items.each(&:destroy!)
+      distribution.line_items.delete_all
```

Comment:
So we _want_ to skip the callbacks?

Reply:
Well,  if we leave it with the callbacks skipped,  maybe we should put a big ol' comment on LineItem that there is code that is dependent on it not having call backs.

Reply:
(because we love our future developers)

Reply:
The argument against doing reload is performance, I presume?

Reply:
Hmm... more that even with the reload there's a small chance of a race condition, whereas `delete_all` goes right to the database and _should_ make it a lot less likely.

Reply:
Ok.  I'll throw a comment on LineItem.    Perhaps our long term solution will not need to have that caution,  but in the meantime....